migrations/versions/830_add_supplierframework_prefill_declaration.py

```diff
@@ -0,0 +1,26 @@
+"""Add SupplierFramework.prefill_declaration_from_framework_id column and constraints
```

Comment:
The migration part of this should go in its own separate pull request and get released first on its own.